rowbinary/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+use crate::data_types::{Column, DataTypeNode};
+use crate::decoders::decode_string;
+use crate::error::ParserError;
+use crate::leb128::decode_leb128;
+
+pub mod data_types;
+pub mod decoders;
+pub mod error;
+pub mod leb128;
+
+pub fn parse_rbwnat_columns_header(bytes: &mut &[u8]) -> Result<Vec<Column>, ParserError> {
```

Comment:
probably should also expose a function that parses a single type from a string, so it is also useful for Native